packages/maskbook/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx

```diff
@@ -322,103 +367,109 @@ export function ClaimAllDialog(props: ClaimAllDialogProps) {
+                                {swappedTokensOld.length > 0 ? (
+                                    <div className={classes.content}>
+                                        {swappedTokens.length > 0 && swappedTokensOld.length > 0 ? (
+                                            <Typography color="textPrimary" className={classes.contentTitle}>
+                                                {t('plugin_ito_claim_all_old_contract')}
+                                            </Typography>
+                                        ) : null}
+                                        <Content swappedTokens={swappedTokensOld} />
+                                    </div>
+                                ) : null}
+                                {swappedTokens.length > 0 ? (
+                                    <div className={classes.content}>
+                                        {swappedTokens.length > 0 && swappedTokensOld.length > 0 ? (
+                                            <Typography color="textPrimary" className={classes.contentTitle}>
+                                                {t('plugin_ito_claim_all_new_contract')}
+                                            </Typography>
+                                        ) : null}
+                                        <Content swappedTokens={swappedTokens} />
+                                    </div>
+                                ) : null}
+                            </>
+                        ) : !showNftAirdrop ? (
+                            <div className={classes.emptyContentWrapper}>
+                                <Typography color="textPrimary">{t('plugin_ito_no_claimable_token')} </Typography>
+                            </div>
+                        ) : null}
+                        {(swappedTokens && swappedTokens.length > 0) ||
+                        (swappedTokensOld && swappedTokensOld.length > 0) ||
+                        showNftAirdrop ? (
+                            <div className={classes.actionButtonWrapper}>
+                                <EthereumChainBoundary
+                                    chainId={chainId}
+                                    noSwitchNetworkTip={true}
+                                    switchButtonStyle={{
+                                        backgroundColor: '#1C68F3',
+                                        '&:hover': {
+                                            backgroundColor: '#1854c4',
+                                        },
+                                        minHeight: 'auto',
+                                        width: '100%',
+                                        fontSize: 18,
+                                        fontWeight: 400,
+                                    }}>
+                                    {(swappedTokens && swappedTokens.length > 0) ||
+                                    (swappedTokensOld && swappedTokensOld.length > 0) ? (
```

Comment:
```suggestion
                                    {swappedTokens?.length || swappedTokensOld?.length ? (
```

Reply:
Object is possibly 'undefined'.ts(2532)
const swappedTokens: SwappedToken[] | undefined